pkg/login/social/github_oauth.go

```diff
@@ -70,6 +71,10 @@ func NewGitHubProvider(settings map[string]any, cfg *setting.Cfg, features *feat
 		// skipOrgRoleSync: info.SkipOrgRoleSync
 	}
 
+	if len(teamIdsSplitted) != len(teamIds) {
+		provider.log.Warn("Failed to parse team ids. Team ids must be a list of numbers.", "teamIds", teamIdsSplitted)
+	}
```

Comment:
nit: I would probably move this up to line 62, just so that it's easier to read the code and see that this case is handled with a warning log.

Reply:
I put this check here because the `provider` (and its `logger`) is initialized in line#64 and I didn't want to initialize a logger/change the structure for this.

Reply:
That makes sense, thanks for explaining your thinking.